frontend/src/helpers/helpers.ts

```diff
@@ -1,2 +1,7 @@
 export { configureString } from 'shared/build/index.js';
 export { getAvatarInitials } from 'shared/build/index.js';
+export {
+    type ChatListItemType,
```

Comment:
Have I properly added type from shared?

Reply:
```
export {
    type ChatListItemType,
    configureString,
    getAvatarInitials,
    getItemsWithSelected,
    getSearchedItems,
} from 'shared/build/index.js';
```

Reply:
initialy it was exported on separate way, so I decided not to change